packages/visx-tooltip/src/tooltips/Tooltip.tsx

```diff
@@ -39,8 +39,11 @@ export default function Tooltip({
     <div
       className={cx('visx-tooltip', className)}
       style={{
-        top: top == null || offsetTop == null ? top : top + offsetTop,
-        left: left == null || offsetLeft == null ? left : left + offsetLeft,
+        left: 0,
+        top: 0,
+        transform: `translate(${
+          left == null || offsetLeft == null ? left ?? 0 : left + offsetLeft
+        }px, ${top == null || offsetTop == null ? top ?? 0 : top + offsetTop})`,
```

Comment:
oops forgot a unit here, I may just commit this to get it merged

```suggestion
        }px, ${top == null || offsetTop == null ? top ?? 0 : top + offsetTop}px)`,
```